### DISCUSSION TITLE
Get from Map current position a BBOX & Center Point Lat Long and Zoom Level

### DISCUSSION THREAD

**Discussion by geospatialmapmashup** [TerriaJS/terriajs#5115] [Q&A]
is there any function to get location of map 
BBOX/Bounds/bounding box
or Center Point Lat Long and Zoom Level?

as we are packaging TerriaJs/TerriaMap into Mobile iOS and Android and Windows Apps http://earthexplorer.world
we need hooks /bridge from webview javascript to native code

**Comment by ahmednoureldeen** [comment]
+1
